Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM alpine:3.2
+
+RUN apk add --update \
+    php-fpm \
+    php-mcrypt \
+    php-curl \
+    php-openssl \
+    php-phar \
+    php-ctype \
+    php-json \
+    curl \
+    git \
+    php-dom \
+    alpine-sdk \
+    php-dev \
+    autoconf \
+    openssl-dev \
+    php-pdo \
+    php-pdo_pgsql \
+    php-pdo_odbc \
+    php-pdo_mysql \
+    php-pdo_sqlite \
+    php-opcache && \
+    sed -i 's/\;date\.timezone\ \=/date\.timezone\ \=\ Europe\/Berlin/g' /etc/php/php.ini && \
```

Comment:
> @Kapeli do we care about timezone at all? 

No idea. My server doesn't have a timezone set in php.ini. I don't think Lumen cares about the php.ini timezone anyways. The timezone is either set using `APP_TIMEZONE` in `env` or it defaults to UTC.

Timezones don't currently matter as I'm not displaying dates anywhere, but they might in the future. Might be best to keep them UTC just in case. Not a huge issue though as I'll most likely take into account different timezone settings when adding time-related stuff.